infra/Pos/KnownPeers.hs

```diff
@@ -28,3 +31,10 @@ instance {-# OVERLAPPABLE #-}
     ( Monad m, MonadTrans f, MonadFormatPeers m ) =>
         MonadFormatPeers (f m) where
   formatKnownPeers k = lift (formatKnownPeers k)
+
+class HasOutboundQ m where
+  type Pack m :: * -> *
```

Comment:
Not such a great name in my opinion. Would it be workable to get rid of this associated type and have

`getOutboundQ :: m (OutboundQ (EnqueuedConversation m) NodeId Bucket)`

?